wtxmgr/tx.go

```diff
@@ -834,7 +834,7 @@ func (s *Store) UnspentOutputs(ns walletdb.ReadBucket) ([]Credit, error) {
 		rec, err := fetchTxRecord(ns, &op.Hash, &block)
 		if err != nil {
 			return fmt.Errorf("unable to retrieve transaction %v: "+
-				"%v", op.Hash, err)
+				"%", op.Hash, err)
```

Comment:
Should be `%w`. 

We also need to tag this module after merge to then update the main module.

Reply:
Oops, nice catch. Fixed.